libraries/xcode.rb

```diff
@@ -50,7 +50,7 @@ def current_path
     def installed?
       return false if installed_path.nil?
 
-      installed_path.any?
+      installed_path.any? && shell_out('/usr/bin/xcodebuild -showsdks').exitstatus == 0
```

Comment:
/usr/bin/xcodebuild is not a perfect standin for the health of Xcode.
1) xcodebuild will run against whatever the developer dir is / whatever `xcode-select --print-path` is pointing at. If that isn't pointing at your new Xcode, you'll get the wrong result.
2) This doesn't handle the situation that someone is installing multiple builds of Xcode because you can have more than one Xcode but you'll only have one `xcodebuild`. Not sure if that's a common case

Reply:
```suggestion
      installed_path.any? && shell_out("DEVELOPER_DIR=#{installed_path} /usr/bin/xcodebuild -showsdks").exitstatus == 0
```